lib/phoenix_profiler/toolbar/toolbar_live.ex

```diff
@@ -65,9 +65,19 @@ defmodule PhoenixProfiler.ToolbarLive do
   end
 
   defp assign_toolbar(socket, profile) do
-    %{metrics: metrics} = profile
+    %Profile{data: %{metrics: metrics}} = profile
+
+    socket =
+      case profile.data[:exception] do
```

Comment:
Seems like there's a race condition here, the toolbar is mounted before telemetry has a chance to collect the event ` [:phoenix_profiler, :endpoint, :exception]` so `profile.data[:exception]` is always null at this point.

I'm assuming that http://localhost:4000/errors/assign-not-available should populate `@exits` considering https://github.com/mcrumm/phoenix_profiler/blob/ae6ca03ee8237e51844d099562baaf5888ec33c4/lib/phoenix_profiler/endpoint.ex#L55 and https://github.com/mcrumm/phoenix_profiler/blob/ae6ca03ee8237e51844d099562baaf5888ec33c4/lib/phoenix_profiler/telemetry.ex#L43